pkg/deployment/inventory.go

```diff
@@ -51,6 +51,8 @@ func GenerateNodeSetInventory(ctx context.Context, helper *helper.Helper,
 		nodeSetGroup.Vars["edpm_tls_certs_enabled"] = "true"
 	}
 
+	nodeSetGroup.Vars["ansible_ssh_private_key_file"] = fmt.Sprintf("/runner/env/ssh_key/ssh_key_%s", instance.Name)
```

Comment:
could we have this fmt.Sprintf in a func in nodeset_types, since it needs to be done twice

Reply:
Sorry, I've just noticed this comment. Where is the other one?